environment_setup/Dockerfile

```diff
@@ -4,11 +4,20 @@ LABEL org.label-schema.vendor = "Microsoft" \
       org.label-schema.url = "https://hub.docker.com/r/microsoft/mlopspython" \
       org.label-schema.vcs-url = "https://github.com/microsoft/MLOpsPython"
 
-       
+# Utilities for package installation
+RUN apt-get update && apt-get install -y \
+    curl \
+    apt-transport-https \
+    gcc
 
-COPY environment_setup/requirements.txt  /setup/
-	
-RUN apt-get update && apt-get install gcc -y && pip install --upgrade -r /setup/requirements.txt && \ 
-    conda install -c r r-essentials
+# Install dotnet runtime used to generate code coverage report
+RUN curl -O https://packages.microsoft.com/config/ubuntu/19.04/packages-microsoft-prod.deb \
+    && dpkg -i packages-microsoft-prod.deb \
+    && apt-get update && apt-get install -y dotnet-runtime-3.1
```

Comment:
Do we really need `dotnet-runtime-3.1`?  I assume code coverage will still show up in Azure DevOps, but we'll just get the warning of no auto-generated html reports, no?

Reply:
Without dotnet runtime we only get the report as an artifact.

With dotnet runtime we get an an embedded HTML report

![image](https://user-images.githubusercontent.com/22099722/73478431-5d700800-4396-11ea-871c-0d09f3824d57.png)

this can be achieved by:
1. Adding dotnet runtime to mlopsython container, is an additional 104 MB installation.
2. [install dotnet](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops) at pipeline execution time.

Given the impact of 1 or 2 in terms of build time and the limited benefit of the embedded report, I will update the PR to go for neither option.

Reply:
Another idea would be to add 2. commented out in the pipeline if people want to enable the code coverage html report

Reply:
OK. Note that we will see this warning:

![image](https://user-images.githubusercontent.com/22099722/73482464-184fd400-439e-11ea-810c-6419c4b9254c.png)

Reply:
👌

Reply:
Fixed. There is no update to the Dockerfile anymore, but the mlopspython image must still be rebuilt because of updates to requirements.txt.
I tested installing dotnet core but coverage report generation then fails with a missing libicu library, and sudo is not available to install it. I suggest investigating this at a later point.